docs/3-retrieving-data-as-lists.md

```diff
@@ -54,21 +60,23 @@ const queryList = af.database.list('/items', {
 To get the list in realtime, create a list binding as a property of your component or service.
 Then in your template, you can use the `async` pipe to unwrap the binding.
 
+Update `/src/app/app.component.ts` to import `FirebaseListObservable` from angularfire2 and iterate thru the list once data is retrieved. Also note the change in attribute `templateUrl` to inline `template` below.
```

Comment:
@katowulf Changing this from external Url to inline template is just to keep the flow of the tutorial consistent across all the sections. Since in the initial section, the changes were done to an inline template. Happy to revert the changes as per your steer. 

Note:- I am also planning to make changes to section 4 and 5 of the tutorial to keep it consistent, while making other changes.